server/src/services/folders.service.ts

```diff
@@ -19,9 +19,15 @@ export class FoldersService {
     @InjectRepository(App)
     private appsRepository: Repository<App>,
     private usersService: UsersService
-  ) {}
+  ) { }
 
   async create(user: User, folderName): Promise<Folder> {
+    if (folderName.length < 1) {
+      throw new BadRequestException('Folder name cannot be shorter than 1 character');
```

Comment:
Shall we create a DTO([example](https://github.com/ToolJet/ToolJet/blob/develop/server/src/dto/version-create.dto.ts)) and add `@maxLength` validation?

Reply:
Could that be in a new issue? I'm running out of time for Hacktoberfest.